libnavigation-core/src/main/java/com/mapbox/navigation/core/routerefresh/RouteRefreshController.kt

```diff
@@ -85,43 +147,57 @@ internal class RouteRefreshController(
                 """.trimIndent(),
                 LOG_CATEGORY
             )
+            null
         }
     }
 
-    private fun createRouteRefreshCallback(
+    private fun logRoutesDIff(
+        newRoute: NavigationRoute,
         oldRoute: NavigationRoute,
         currentLegIndex: Int,
-        onRoutesRefreshed: (List<NavigationRoute>) -> Unit
-    ) = object : NavigationRouterRefreshCallback {
+    ) {
+        logI("Successful route refresh", LOG_CATEGORY)
```

Comment:
nit: Move this log to the invocation point, just the fact of comparing routes doesn't not mean the refresh was successful from this function's perspective.